extensions/interactions/ItemSelectionInput/static/item_selection_input.css

```diff
@@ -36,6 +36,10 @@ md-checkbox.item-selection-input-checkbox.md-default-theme.md-checked .md-icon {
   border-bottom: 0px;
 }
 
+.item-selection-input-container img {
+  max-height: 200px;
```

Comment:
For this part, don't you need to set max-width (so it doesn't spill out of the card)?

Reply:
Hmm, in the images I've tried it doesn't overflow the width. At least in FF, that seems to be enforced by [this](https://github.com/oppia/oppia/blob/c69ecfb31503400c26797667b8d7e308b7f0562e/extensions/interactions/ItemSelectionInput/static/item_selection_input.css#L49).

It looks like the card itself can get cut off if the screen is too small, but that's a separate issue.